modules/op-using-custom-scc-to-improve-prod-security.adoc

```diff
@@ -0,0 +1,19 @@
+[id='op-using-custom-scc-to-improve-prod-security_{context}']
+= Using a custom SCC to improve pod security
+
+You can improve security by using a custom security context constraints (SCCs) because it's easy to migrate pipelines that need to be run or built as root. To further lock down your installation, allow the container to run only as user 1000 when the pod asks for it and the default root is not allowed.
```

Comment:
```suggestion
Because you can migrate pipelines that must be run or built as root, you can improve security by using custom security context constraints (SCCs). To lock down your installation further when the default root user is not allowed, allow the container to run only as user `1000` when the pod requests what permissions to use.
```
- delete indefinite article for plural noun
- do not use contractions
- per the ISG's glossary, avoid using the word "easy" or including any opinionated phrases like "it is easy to".
- The meaning of this paragraph is not entirely clear to me. I've tried to rephrase the sentences for clarity, but please review for accuracy, but I still don't understand what an SCC is and how the second sentence using an SCC to set permissions. I'd suggest expanding this paragraph to clarify the relationship between an SCC and setting the permission of the container to user `1000`.